pkg/bitcoin_difficulty/chain.go

```diff
@@ -0,0 +1,38 @@
+package btcdiff
+
+import (
+	"github.com/keep-network/keep-core/pkg/bitcoin"
+	"github.com/keep-network/keep-core/pkg/chain"
+)
+
+// Chain defines the interface that pertains specifically to the process of
+// updating Bitcoin difficulty.
+type Chain interface {
```

Comment:
How/where can we check what should be included "specifically to the process of updating Bitcoin difficulty"?

Reply:
Bitcoin difficulty updating means updating the [LightRelay](https://github.com/keep-network/tbtc-v2/blob/8b9d2629bf4333e650a54f32a4da7cf86bf6785e/solidity/contracts/relay/LightRelay.sol#L83) contract.

The most important function there is `Retarget` which does the actual updating, but there are also some getters that we must use before we can update the `LightRelay`.
Here is [the PR which adds the updating](https://github.com/keep-network/keep-core/pull/3394).